### ISSUE TITLE
multi-line promise toasts change height on hover when using expand

### ISSUE THREAD

**Issue by ThaUnknown** [wobsoriano/svelte-sonner#13]
```jsx
<Toaster expand={true} />
```
```js
toast.promise(Promise.resolve(), { success: 'This my very very very very very very very very very very very very very very very very very very very very long and descriptive promise toast!!!' })
```
when hovering over these toasts, they will change height as their top paddings will disappear, causing weird layout shifts, it looks very ugly

**Comment by ThaUnknown** [comment]
this has been fixed

**Comment by TomRadford** [comment]
+1 on this for me. Will be awesome to get this one fixed since this is such a beautiful modal implementation.

**Comment by wobsoriano** [comment]
WIll look into it this week

**Comment by wobsoriano** [comment]
Hey y'all, can any of you provide a demo of this? I can't replicate / notice the shift

**Comment by ThaUnknown** [comment]
@wobsoriano hey sorry, this error report is inaccurate, the reproduction steps are actually as follows:
```html
<Toaster expand={true} />
```

```js
toast.promise(Promise.resolve(), { success: 'This my very very very very very very very very very very very very very very very very very very very very long and descriptive promise toast!!!' })
```
it happens only with promise toasts, not normal toasts

**Comment by wobsoriano** [comment]
Are you still getting it with the latest version @ThaUnknown?

**Comment by TomRadford** [comment]
@wobsoriano seem like I'm still getting this issue on `0.3.6` sadly when there's only one toast item:

https://github.com/wobsoriano/svelte-sonner/assets/7515754/34084531-140d-4442-8cc4-da0e65a22c6e

**Comment by huntabyte** [comment]
I'm not able to reproduce with the latest version, is anyone able to reproduce or should this be closed?

**Comment by ThaUnknown** [comment]
I can still reproduce this, I'll make some minimal test case later

**Comment by ThaUnknown** [comment]
[repro.zip](https://github.com/wobsoriano/svelte-sonner/files/13796712/repro.zip)

this issue happens when the error/success has more lines than the pending text, I've attached a stupidly simple reproduction case, which runs on body click

**Comment by wobsoriano** [comment]
@ThaUnknown can you use https://stackblitz.com or https://codesandbox.io/ to create a repro?

**Comment by ThaUnknown** [comment]
you can simply paste the files into stack blitz yourself

**Comment by jakob-kruse** [comment]
I can still repro this, too

https://stackblitz.com/edit/vitejs-vite-gtdpxr?file=src%2FApp.svelte

**Comment by huntabyte** [comment]
Thanks for the repro @jakob-kruse, will look into this!

**Comment by huntabyte** [comment]
This definitely helps!

**Comment by Chiroyce1** [comment]
This is a very beautiful tool and certainly elevates any Svelte app's experience! 10/10 except for this one small issue, is there a temporary fix that we can use?

**Comment by wobsoriano** [comment]
@jakob-kruse thanks for the repro! It's now fixed in 0.3.19

https://stackblitz.com/edit/vitejs-vite-d7j7sy?file=package.json

**Comment by AlexDemzz** [comment]
Its fixed in the 0.3.19 but the bug get back in 0.3.28

**Comment by cornzz** [comment]
@wobsoriano Hi, we are also having this problem.

We are using v0.3.28, since we are on svelte 4 and the library seems incompatible with that since 1.0.0. Everything is working great except this small issue where promise toasts, which changed height after the promise resolves, jump up when being hovered. We are using a custom notification component and implement promise handling ourself, where `toast.create` isn't actually called again after the promise resolves, but I also tried that and it didn't fix the problem.

https://github.com/user-attachments/assets/70db7a59-43fa-44b1-a381-83c040843ba4
